src/atomate2/lobster/schemas.py

```diff
@@ -149,6 +151,18 @@ class CondensedBondingAnalysis(BaseModel):
         description="Stores the COHP plot data based on relevant bond labels "
         "for site as keys",
     )
+    whichbonds: str = Field(
```

Comment:
_ ... 😉

Reply:
Ops. Sorry will fix this. I by mistake ended up merging it